integrations/GA4/browser.js

```diff
@@ -260,6 +260,10 @@ export default class GA4 {
   }
 
   getDataForIntegrationsObject() {
-    return getGa4SessionId(this.measurementId);
+    return {
+      GA4: {
```

Comment:
Is `GA4` the display name? We should use display name.

Reply:
updated